src/spec-node/featuresCLI/test.ts

```diff
@@ -17,8 +17,9 @@ export function featuresTestOptions(y: Argv) {
 			'skip-scenarios': { type: 'boolean', default: false, description: 'Skip all \'scenario\' style tests.  Cannot be combined with \'--global--scenarios-only\'.' },
 			'skip-autogenerated': { type: 'boolean', default: false, description: 'Skip all \'autogenerated\' style tests.' },
 			'base-image': { type: 'string', alias: 'i', default: 'ubuntu:focal', description: 'Base Image. Not used for scenarios.' },  // TODO: Optionally replace 'scenario' configs with this value?
-			'remote-user': { type: 'string', alias: 'u', default: 'root', describe: 'Remote user.  Not used for scenarios.', },  // TODO: Optionally replace 'scenario' configs with this value?
+			'remote-user': { type: 'string', alias: 'u', describe: 'Remote user.  Not used for scenarios.', },  // TODO: Optionally replace 'scenario' configs with this value?
 			'log-level': { choices: ['info' as 'info', 'debug' as 'debug', 'trace' as 'trace'], default: 'info' as 'info', description: 'Log level.' },
+			'apply-image-dev-container-metadata': { type: 'boolean', default: true, description: 'Apply image metadata to the resultant image appended by previously building with the devcontainer CLI' },
```

Comment:
Would `--skip-image-metadata` align better with the other `skip` options above?

Reply:
Good point, I like that better.